### PR TITLE
Add columnLabel to property

### PR DESCRIPTION
**Pull request by sarthak-marwaha**
- Added columnLabel to properties
- Column headers now use 'columnLabel' when available
- Column tooltips still use long form 'label' for a property

Screen Shot 2021-05-07 at 5 01 47 PM
Screen Shot 2021-05-07 at 5 01 27 PM

### CODE REVIEW

File: src/foam/u2/view/TableCellFormatter.js [hunk 1]
```diff
@@ -75,6 +75,11 @@ foam.CLASS({
     {
       class: 'Int',
       name: 'tableWidth'
+    },
+    {
+      class: 'String',
+      documentation: 'Column label that overrides the label property in table headers',
+      name: 'columnLabel'
```
Comment (kgrgreer) [inline src/foam/u2/view/TableCellFormatter.js:82]: The expression: for this should just return the regular label. That would simplify line 60 above.

File: src/foam/nanos/column/ColumnConfigPropertyValue.js [hunk 2]
```diff
@@ -64,33 +64,37 @@ foam.CLASS({
       code: function(of, propName) {
         var cls = of;
         var axiom;
-        var columnHeader = [];
+        var colPath = []; // Array with full path to column header
+        var colLabel = ''; // Returns columnLabel property if applicable
```
Comment (olhabn) [inline src/foam/nanos/column/ColumnConfigPropertyValue.js:69]: returnColumnHeader used in couple of different places for example in `src/foam/nanos/export/GoogleSheetsOutputter.js`.

so we'll need to do            ` columnLabel: columnConfig.returnColumnHeader(of, propName)[1],` on line 89`src/foam/nanos/export/GoogleSheetsOutputter.js`.

File: src/foam/nanos/column/CommonColumnHandler.js [hunk 3]
```diff
@@ -59,7 +59,7 @@ foam.CLASS({
       return this.returnAxiomHeader(this.returnNElementIfArray(rootProperty, 0));
     },
     function returnAxiomHeader(axiom) {
-      return axiom.tableHeader  ? axiom.tableHeader() : axiom.label;
+      return axiom.columnLabel || axiom.label;
```
[the remark this answers is not in the document]
Comment (sarthak-marwaha) [reply src/foam/nanos/column/CommonColumnHandler.js:62]: The editColumnView beaks if I do that. It stops showing labels for actions. Can't figure out why. @olhabn got any insight?

File: src/foam/u2/view/TableCellFormatter.js [hunk 4]
```diff
@@ -88,7 +89,7 @@ foam.CLASS({
 
   properties: [
     {
-      name: 'tableHeader',
+      name: 'columnLabel',
```
[the remark this answers is not in the document]
Comment (sarthak-marwaha) [reply src/foam/u2/view/TableCellFormatter.js:92]: Oversight, will fix

File: src/foam/nanos/column/CommonColumnHandler.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (olhabn) [reply src/foam/nanos/column/CommonColumnHandler.js:62]: does Action have tableCellFormatter?

File: src/foam/u2/view/TableCellFormatter.js [hunk 5]
```diff
@@ -57,7 +57,7 @@ foam.CLASS({
     {
       name: 'tableHeader',
```
[the remark this answers is not in the document]
Comment (sarthak-marwaha) [reply src/foam/u2/view/TableCellFormatter.js]: This is from an outdated commit. The new commit removes this.

**Comment by sarthak-marwaha** [comment]
Update: Fixed CSV export and merge conflict

**Pull request merged**